environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentCreationDto.java

```diff
@@ -0,0 +1,166 @@
+package com.sequenceiq.environment.environment.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.commons.collections4.CollectionUtils;
+
+import com.sequenceiq.environment.api.v1.environment.model.request.CredentialAwareEnvRequest;
+import com.sequenceiq.environment.network.dto.NetworkDto;
+
+public class EnvironmentCreationDto {
```

Comment:
I think as our service layer hasn't been refactored to use the `JSON->Dto->Entity` conversion logic we could get rid of these "Dto" classes for now. It would caused missed convention within environment service.

Reply:
I disagree, we can start refactoring as we develop it.